common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3273,11 +3273,11 @@ public static enum ConfVars {
 
     HIVE_COMPACTOR_ABORTEDTXN_THRESHOLD("hive.compactor.abortedtxn.threshold", 1000,
         "Number of aborted transactions involving a given table or partition that will trigger\n" +
-        "a major compaction."),
+        "a major compaction / cleanup of aborted directories."),
```

Comment:
would it actually trigger compaction?

Reply:
This config is used for controlling the number of aborted txns after which the cleanup is triggered (either cleanup / compaction) depending on what is expected.

Reply:
we are not doing compaction for aborts, just cleanup